src/components/MovieEditPopUp.js

```diff
@@ -0,0 +1,112 @@
+import React from "react";
+import Styled from "styled-components";
+
+const StyledInputBlock = Styled.div`
+display: flex;
+flex-direction: column;
+margin: 15px 0 0;
+`;
+
+const Styledlabel = Styled.label`
+color: #f65261;
+font-size: 18px;
+margin: 0 0 10px;
+`;
+
+const StyledInput = Styled.input`
+border-radius: 6px;
+background-color: #424242;
+border: none;
+font-size: 16px;
+padding: 10px;
+color: #fff;
+`;
+const StyledSelect = Styled.select`
+border-radius: 6px;
+background-color: #424242;
+border: none;
+font-size: 16px;
+padding: 10px;
+color: #fff;
+`;
+const StyledButtonBlock = Styled.div`
+display: flex;
+margin: 30px 0 0;
+justify-content: flex-end;
+`;
+const StyledReset = Styled.input`
+border-radius: 6px;
+background-color: transparent;
+border: 2px solid #f65261;
+font-size: 16px;
+padding: 15px 10px;
+margin: 0 15px 0 0; 
+color: #f65261;
+width: 150px;
+cursor: pointer;
+&::placeholder {
+  color: #f65261;
+}
+`;
+const StyledSubmit = Styled.input`
+border-radius: 6px;
+background-color: #f65261;
+border: none;
+font-size: 16px;
+padding: 15px 10px;
+color:#fff;
+width: 150px;
+cursor: pointer;
+`;
+const StyledId =  Styled.h5`
+margin: 10px 0;
+`
+
+const MovieEditPopUp = (props) => {
+  return (
+    <>
+      <h1>EDIT MOVIE</h1>
+      <form>
+      <StyledInputBlock>
+          <Styledlabel>MOVIE ID</Styledlabel>
+          <StyledId>{props.id}</StyledId>
+        </StyledInputBlock>
+        <StyledInputBlock>
+          <Styledlabel>TITLE</Styledlabel>
+          <StyledInput type="text" name="title" placeholder="Moana"/>
```

Comment:
why don't you use a movie properties to fill in the form fields? Passing the whole movie object to the component of course